app/models/card.rb

```diff
@@ -25,19 +24,16 @@ def check_translation(user_translation)
     sm_hash = SuperMemo.algorithm(interval, repeat, efactor, attempt, distance, 1)
 
     if distance <= 1
-      sm_hash.merge!({ review_date: Time.now + interval.to_i.days, attempt: 1 })
-      update(sm_hash)
+      update(sm_hash.merge!({ review_date: Time.now + interval.to_i.days, attempt: 1 }))
```

Comment:
Line is too long. [88/80]<br>Redundant curly braces around a hash parameter.<br>Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.